graphs/DataVisualisation/index.js

```diff
@@ -56,9 +59,15 @@ module.exports = React.createClass({
         totalFormat    = props.totalFormat,
         title          = props.title,
         valueConverter = props.valueConverter,
-        loading        = props.loading;
+        loading        = props.loading,
+        emptyState     = props.emptyState;
 
-    if (loading || typeof(total) != 'undefined') {
+    if (emptyState) {
+      return (
+        <SingleNumber title={ title } format={ totalFormat } value={ valueConverter(total) } emptyState={ emptyState }/>
+      );
+    }
+    else if (loading || typeof(total) !== 'undefined') {
```

Comment:
You still don't need this else. Just the if. The above if will return the function early if true.